classes/04class/exercises/c04-iac02/README.md

```diff
@@ -0,0 +1,34 @@
+# VPC (c04-iac02)
+
+In this exercise, you'll need to create a new VPC called `devopsacademy-iac`, but this time, every resource of this VPC will be created using Terraform. ALl your resources must be created in a single `vpc.tf` file.
```

Comment:
reference to past exercise where they have done the same on the console

Reply:
added